src/Axiom/UniquenessOfIdentityProofs.agda

```diff
@@ -59,19 +62,14 @@ module Constant⇒UIP
 -- function over proofs of equality which is constant: it returns the
 -- proof produced by the decision procedure.
 
-module Decidable⇒UIP
-  {a} {A : Set a} (_≟_ : Decidable {A = A} _≡_)
-  where
+module Decidable⇒UIP (_≟_ : Decidable {A = A} _≡_) where
 
   ≡-normalise : _≡_ {A = A} ⇒ _≡_
```

Comment:
Would these proofs yield (more easily?) to `recompute` after #2243 ?

Reply:
Because `recompute-irr` witnesses the irrelevance of proofs computed using `recompute` ;-)